### PR TITLE
Chat commands for private messages, replying and unprivate.

### PR DESCRIPTION
**Pull request by FedexCain**
Hey, 
I noticed while playing that I normally don't move my hands from the keyboard. If someone sends you a private message the only method to privately write them back is to move one's hand to the mouse and click their username. This is especially tedious while conversing with multiple users that way. 

The commands "/private \<username\>", "/reply" and "/unprivate" allow you to initiate and end private messages with other people in the room.

I hope you consider merging this :)

### CODE REVIEW

File: public/js/app.js [hunk 1]
```diff
@@ -517,7 +519,12 @@
 
     if (to) {
       // Private Message
-      prefix = '(' + (nickname === from ? 'To ' + to : 'From ' + prefix) + ')';
+      if(nickname === from ) {
```
Comment (lpinca) [inline public/js/app.js]: Add a space after this `if` and all the others to keep a consistent style.

File: public/js/app.js [hunk 2]
```diff
@@ -517,7 +519,12 @@
...
+        prefix = '(To ' + to + ')';
+      } else {
```
Comment (lpinca) [inline public/js/app.js]: Put the `else` branch on a new line for the same reason stated above.

File: public/js/app.js [hunk 3]
```diff
@@ -815,14 +822,14 @@
         $messagebox.val('');
 
         if (value) {
-          if (pvtmsgto) {
```
Comment (lpinca) [inline public/js/app.js]: I didn't want to parse slash commands in private message, but i can live with it.

File: public/js/app.js [hunk 4]
```diff
@@ -951,6 +958,28 @@
...
+  };
+
+  // Reply to the last player who sent you a private message
+  var replyToPrivate = function(args, $outcome) {
+    if(!replyto) {
+      $outcome.text('(From binb): There\'s noone to reply to.');
+      return addChatEntry($outcome);
+    }
+    if($.inArray(replyto, usersnicks) === -1) {
+      $outcome.text('(From binb): They aren\'t here anymore.');
```
Comment (lpinca) [inline public/js/app.js]: Can we use a better message here? I see no reason for the plural since we are targeting a single player.

Comment (FedexCain) [inline public/js/app.js]: I tried to stay gender neutral. I thought that's how you do it in english (sorry non native speaker). I'll just change it so it uses the name of the user in the replyto variable.

Comment (lpinca) [inline public/js/app.js]: Something like `The player is no longer here` should work.

File: public/js/app.js (hunk 2 above)
Comment (FedexCain) [inline public/js/app.js]: fixed in 3c16d27

File: public/js/app.js (hunk 4 above)
Comment (FedexCain) [inline public/js/app.js]: Is 391fee5 okay, or would you like me to change it to your suggestion?

**Comment by lpinca** [comment]
Merged ad 816dcb00914c227a13b40105d209d6b81003064f. Thank you very much!

**Pull request closed without merging**